pkg/controller/compute/eks.go

```diff
@@ -61,10 +61,14 @@ const (
 )
 
 var (
-	log           = logging.Logger.WithName("controller." + controllerName)
-	ctx           = context.Background()
-	result        = reconcile.Result{}
-	resultRequeue = reconcile.Result{Requeue: true}
+	log = logging.Logger.WithName("controller." + controllerName)
+	ctx = context.Background()
+)
+
+// Amounts of time we wait before requeuing a reconcile.
+const (
+	aShortWait = 30 * time.Second
+	aLongWait  = 60 * time.Second
```

Comment:
not a blocker for this pr, but I think these consts should be in the runtime library.

Reply:
I see these as short term solutions, with the long term solution being using the managed resource reconciler to implement this controller.

https://godoc.org/github.com/crossplaneio/crossplane-runtime/pkg/resource#WithShortWait

I believe these were global constants for a while, until someone correctly pointed out that 30 and 60 seconds aren't necessarily the best requeue durations for all managed resource controllers. Instead now we make them the default, but allow controllers to override them if appropriate.